spec/system/task_spec.rb

```diff
@@ -85,8 +84,7 @@
 
       it '既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと' do
         # TODO: FactoryBotのtraitを利用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday)
+        task = FactoryBot.create(:task, :done, project: project)
```

Comment:
traitを使って作成するデータを変数に入れる場合は、変数名をわかりやすくした方が良いですね。
```suggestion
        done_task = create(:task, :done)
```

Reply:
68ac349